### ISSUE TITLE
Keep being disconnected

### ISSUE THREAD

**Issue by Nojarosann** [TheLastBen/fast-stable-diffusion#2115]
I keep being disconnected. I don't think it's my internet because i keep testing it and download and upload speeds are both fine, the same speed as it was when the tnotebook was working normally.

But now it just keeps disconnecting me for no reason after i start to run the "Start Stable Diffusion" cell. Ive tried it with multiple alt accounts but the same result. i can barely do 5 images before i get disconnected.

**Comment by sahil-90** [comment]
This problem is not limited to you only, everyone's facing the same damn issue. Seems like google doesn't want people to run the webui for free. I have heard people say it's working just fine for paid users, are you one of them?

**Comment by Nojarosann** [comment]
Nah. Shit this piece of garbage is just so damn expensive with the dolar conversion. I assume there is no other free service to use in this case?

**Comment by sahil-90** [comment]
There's also available a PaperSpace variant of the same repo, did you try using that? It's on the slower side though. Kaggle is yet another platform that might help.

**Comment by Nojarosann** [comment]
i'll give it a try thanks for the help

**Comment by Nojarosann** [comment]
wait how do i run that?

**Comment by maozdemir** [comment]
> This problem is not limited to you only, everyone's facing the same damn issue. Seems like google doesn't want people to run the webui for free. I have heard people say it's working just fine for paid users, are you one of them?

you don't have to hear it from anyone, they officially announced that webui ismonky available on subscriptions.

**Comment by NaveenSohal** [comment]
anyone got solution for this, I don't have good pc to run it locally and I really got into stable diffusion.

**Comment by Wyeel** [comment]
> There's also available a PaperSpace variant of the same repo, did you try using that? It's on the slower side though. Kaggle is yet another platform that might help.

isn't paperspace also paid subscription? And from my research, google colab is the cheapest of the three.

**Comment by maozdemir** [comment]
> anyone got solution for this, I don't have good pc to run it locally and I really got into stable diffusion.

just take a look at the alternatives, some are better than colab machines, either way you gotta psy.

**Comment by iqddd** [comment]
That's lame! I'm already used to unloading on a well-chosen prompt. I don't even know if I can finish with regular stuff.

**Comment by sahil-90** [comment]
https://colab.research.google.com/drive/1wEa-tS10h4LlDykd87TF5zzpXIIQoCmq?authuser=1#scrollTo=8Uvqba0yKW7U | This one's been working just fine for me as of now without any "runtime disconnected" problem. I wonder why some notebooks are having problem while not the others.

**Comment by SDPDVADER** [comment]
![image](https://user-images.githubusercontent.com/123031699/236561544-9c6c9dc1-f446-40f5-a84a-01c9ff02b797.png)
been running for a while.

**Comment by Sooty001** [comment]
> https://colab.research.google.com/drive/1wEa-tS10h4LlDykd87TF5zzpXIIQoCmq?authuser=1#scrollTo=8Uvqba0yKW7U | This one's been working just fine for me as of now without any "runtime disconnected" problem. I wonder why some notebooks are having problem while not the others.

The truth works, it is very strange that the TheLastBen notepad does not work. Although the developers of Google collab wrote that it would no longer be possible to work for free.

**Comment by TheLastBen** [comment]
@Sooty001 they targeted the most used notebook first, the rest will follow

**Comment by SDPDVADER** [comment]
@TheLastBen gotcha.  you still do great work bud.  I appreciate your notebook and everything.

**Comment by ositoMalvado** [comment]
> ![image](https://user-images.githubusercontent.com/123031699/236561544-9c6c9dc1-f446-40f5-a84a-01c9ff02b797.png) been running for a while.

wich notebook? can u share?
